dcpy/connectors/socrata/metadata.py

```diff
@@ -54,7 +54,7 @@ def make_dcp_col(c: pub.Socrata.Responses.Column) -> md.DatasetColumn:
         dcp_col["values"] = [
             {"value": s["item"], "description": FILL_ME_IN_PLACEHOLDER} for s in samples
         ]
-    md.DatasetColumn._validate_data_type = False
+    # md.DatasetColumn._validate_data_type = False # legacy attribute used during migration, no longer there
```

Comment:
Ah - this is why the error is popping up now?

Reply:
Given that this is code to create a placeholder, it seems fine to leave, no? And resolves our issue.

Or maybe it could even go in line 58? Not sure how this arg works. but maybe
```python
md.DatasetColumn(**dcp_col, _validate_data_type = False)
```
would work too?

Reply:
And per the commit message/comment - this I think is explicitly for the purpose of avoiding this error - I don't think it has to do with anything from migration from v1 to v2

Reply:
Yep, this is the reason for the error. If I revert the code back, it should solve the error. Counter q: would we won't this attribute `_validate_data_type` in the parent `Column` obj or `metadata.Column`? 

 Would love to get @alexrichey 's thoughts too on this

Reply:
I don't think we would. We only want to not validate in this case, when we're generating it with placeholders. Other than that, we should always validate at runtime on instantiation

Reply:
Yeah, IIRC this was only for the purpose of spitting out a metadata yaml template with `column.type` of something like "< FILL ME IN! >". I'd just kill every reference to it.

Reply:
What do you mean specifically by kill every reference? Get rid of this util? Per @sf-dcp 's point about if we make this public, I think going from socrata metadata to dcpy metadata is a useful util to have

Reply:
Found this in a github discussion

https://github.com/pydantic/pydantic/blob/6cda388c7a1af29d17973f72dfebd98cff626988/pydantic/main.py#L417

So `Column.construct(**args)` would return an unvalidated column object I think

Reply:
oh, neat. @sf-dcp If 👆works, great. Overall, I'm in favor of whatever is fastest/easiest at the moment

Reply:
I like the idea of replacing all references of `< FILL ME IN! >` with `text`. This way, we keep the integrity while still offering an option to populate a raw template... 

Are we okay with that?

@alexrichey @fvankrieken

Reply:
I think I prefer having the placeholder - it really forces the user to actually enter something explicitly without making any assumptions. But again, just if this command works, otherwise let's just do away with it.

Reply:
let me try the placeholder... will report back

Reply:
Tried the `construct` fn. It's now deprecated and the new [method](https://github.com/pydantic/pydantic/blob/6f31f8f68ef011f84357330186f603ff295312fd/pydantic/main.py#L297) is `model_construct()`. 

Sooo it seems like this method does no validation on input attributes & its values. For example, you can instantiate a model without any keys or random keys like below:

<img width="644" alt="image" src="https://github.com/user-attachments/assets/8c3f6994-aaf4-4d78-94c0-3753abb0c50f">
 
 Using this seems more problematic because we can potentially generate templates with invalid keys... @fvankrieken, I think we should go with the text field

Reply:
Hmm. At least having keys validated would be nice.

I'm not horribly concerned though - this is a util that's designed to create an invalid model, and this seems like a one-liner command to run it without error. Tests can make sure that structure is as expected. In terms of how a user would use it, this seems like less of a source of error to me than autofilling data_types that users might then forget to update. And this is slightly better than say just a dict in that it still shows developers what is expected.

Reply:
Makes sense. Added an assert statement for key validation!